backend/app/views/spree/admin/shared/_refunds.html.erb

```diff
@@ -1,11 +1,11 @@
 <table class="index" id='payments' data-order-id='<%= @order.number %>'>
   <thead>
     <tr data-hook="refunds_header">
-      <th><%= "#{Spree.t('date')}/#{Spree.t('time')}" %></th>
+      <th><%= Spree::Refund.human_attribute_name(:created_at) %></th>
       <th><%= Spree.t(:payment_identifier) %></th>
```

Comment:
`number` already translates to `Identifier` so we could still use that if we're okay changing the visual look or we could add `identifier` as a key in the dictionary although it isn't a database column and just an alias attribute for number.    Or we could have `Spree::Payment.model_name.human Spree::Payment.human_attribute_name(:number)` but that seems pretty terrible.